product/stream_processor/domain_logic/product_notification.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from product.models.products.product import ProductChangeNotification
+from product.stream_processor.integrations.events.event_handler import ProductChangeNotificationHandler
+from product.stream_processor.integrations.events.models.output import EventReceipt
+from product.stream_processor.integrations.events.providers.eventbridge import EventBridge
+
+EVENT_BUS = os.environ.get('EVENT_BUS', '')
```

Comment:
i assume these will get moved to env vars?

Reply:
YUP!! Gonna move all these in the integ/e2e test PR.